src/main/java/io/aiven/klaw/service/EnvsClustersTenantsControllerService.java

```diff
@@ -1245,4 +1211,16 @@ public Map<String, String> getClusterInfoFromEnv(String envSelected, String clus
 
     return clusterInfo;
   }
+
+  public List<Map<String, String>> getSupportedKafkaProtocols() {
+    List<Map<String, String>> supportedProtocols = new ArrayList<>();
+    for (KafkaSupportedProtocol kafkaSupportedProtocol : KafkaSupportedProtocol.values()) {
+      Map<String, String> protocolValues = new HashMap<>();
+      protocolValues.put("name", kafkaSupportedProtocol.getName());
+      protocolValues.put("value", kafkaSupportedProtocol.getValue());
+      supportedProtocols.add(protocolValues);
```

Comment:
What is the contract for `supportedProtocols`? Which maps should be added there?
Is it ok to add NON modifiable maps?
I'm asking because if it is ok to add NON modifiable map then the cycle it could be a bit simplified like
```java
for (KafkaSupportedProtocol kafkaSupportedProtocol : KafkaSupportedProtocol.values()) {
      supportedProtocols.add(
              Map.of("name", kafkaSupportedProtocol.getName(), 
                      "value", kafkaSupportedProtocol.getValue()));
    }
```